deker_server_adapters/utils.py

```diff
@@ -26,8 +28,9 @@ def _request(url: str, node: str, client: Client, request_kwargs: Optional[Dict]
             response = client.get(request_url, **request_kwargs)
         else:
             response = client.get(request_url)
-    except Exception:
-        logger.exception(f"Coudn't get response from {node}")  # noqa
+    except Exception as e:
+        traceback.print_exc()
```

Comment:
May be we want to print just the last exception from trace (`traceback.print_exc(-1)`)?  
It's too loud now.

Reply:
https://github.com/openweathermap/deker-server-adapters/pull/23/commits/23e745a77945bccefa6b7323b89a34652356450e